src/scss/custom/modules/_social-follow.scss

```diff
@@ -0,0 +1,48 @@
+.block-social {
+  text-align: center;
+
+  ul {
+    display: inline-flex;
+    margin: 0;
+
+    @media (max-width: 767px) {
+      flex-flow: wrap;
+    }
+
+    li {
+      display: inline-block;
+      width: 3.5rem;
+      height: 2.5rem;
+      margin: 1rem;
+      cursor: pointer;
+      background-color: $primary;
+      background-repeat: no-repeat;
+      background-position: center;
+      border-radius: 100rem;
+
+      @media (max-width: 767px) {
+        width: 2.5rem;
+        height: 2rem;
+        margin-right: 1.3rem;
+        margin-bottom: 1.3rem;
+        background-size: contain;
+      }
+
+      &:hover {
+        background-color: #24b9d7;
+      }
```

Comment:
I don't like this. `a` is the active element so that's the element that should be styled. You can even skip `cursor: pointer` on it, because it's default. `li` is just the container.

IMO it shouldn't even be `ul` in the first place, because it serves no purpose here. Just a simple `div` with links.

Another problem is the explicit size. The best option here is to use SVGs inlined in the template with `height="1em"` and determine the size with `font-size`. I wouldn't even use `width` or `height` on links. Just `gap` to determine spaces between them. Unless they're going to be styled as buttons with borders, then:
```
a {
  display: inline-flex;
  align-items: center;
  justify-content: center;
  width: 1.5em;
  height: 1.5em;
  border: 1px solid currentcolor;
  // optionally
  border-radius: 50%;
}
```

Reply:
@SharakPL we need to deal with the module, we can't use an inline SVG unfortunately :/

Reply:
> we can't use an inline SVG unfortunately

Why not? I changed my theme to use inline SVG in this module with no problem.

Reply:
Well, I would agree, and I'll push it using your svgs, but it adds some maintenance, as when the module get updated like adding a new social network, the theme needs to deal with it!

Reply:
It's still much easier to simply add another condition and path. Icons are from FontAwesome so if this font would be used, maybe instead of Material Icons, then we could avoid svgs altogether and just use simple classes. Bootstrap Icons also have social media icons, though they don't match as good as those in FA (eg. all squares, like in example above).

Best option would be to not hardcode any social media and just allow user to add any link and pick the icon from FA font. Similar like _Icon_ component in _CreativeElements_ module:
![obraz](https://user-images.githubusercontent.com/5007456/176210132-d4ec6ceb-0260-46cf-8430-38969f2deddc.png)

Reply:
Good point, but I guess we should bump it to a major version, as it's a huge BC for every user of this module

Until it's not done, I guess we can just accept this PR, and then remove it when the module has been updated!